build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml

```diff
@@ -74,6 +74,9 @@ jobs:
         Copy-Item -Path "$targetsFilePath\Intellisense\Microsoft.Windows.ApplicationModel.DynamicDependency.xml" -Destination "$fullpackagePath\lib\uap10.0\Microsoft.Windows.ApplicationModel.DynamicDependency.xml"
         Copy-Item -Path "$targetsFilePath\Intellisense\Microsoft.Windows.PushNotifications.xml" -Destination "$fullpackagePath\lib\uap10.0\Microsoft.Windows.PushNotifications.xml"
 
+        Copy-Item -Path "$targetsFilePath\Intellisense\Microsoft.Windows.AppLifecycle.xml" -Destination "$fullpackagePath\lib\net5.0-windows\Microsoft.Windows.AppLifecycle.Projection.xml"
+        Copy-Item -Path "$targetsFilePath\Intellisense\Microsoft.Windows.ApplicationModel.DynamicDependency.xml" -Destination "$fullpackagePath\lib\net5.0-windows\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.xml"
```

Comment:
>net5.0-windows [](http://example.com/codeflow?start=148&length=14)

why is it net5.0-windows10.0.17763.0 for dll/pdb, but net5.0-windows for xml?

Reply:
Hmm. I see MRT does the versioned dir for Intellisense. Will update to match